service/featuregate/flags.go

```diff
@@ -0,0 +1,99 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package featuregate
+
+import (
+	"flag"
+	"sort"
+	"strings"
+)
+
+const gatesListCfg = "feature-gates"
+
+var gatesList = FlagValue{}
+
+// Flags adds CLI flags for managing feature gates to the provided FlagSet
+func Flags(flags *flag.FlagSet) {
+	flags.Var(
+		gatesList,
+		gatesListCfg,
+		"Comma-delimited list of feature gate identifiers. Prefix with '-' to disable the feature.  '+' or no prefix will enable the feature.")
+}
```

Comment:
I would like to see at least couple of examples for the rules. At least in the func comment.